main.cpp

```diff
@@ -78,19 +78,19 @@ fork.melt();
 //  4)
 //  Noun:       mixer
 //  action 1:   adjusts channel  volume with a fader
-mixer.faderVolume();
+mixer.adjustFaderVolume();
 //  action 2:   enables EQ
-mixer.eqSwitch();
+mixer.enable();

```

Comment:
What does the mixer enable? 
an EQ?
a Send?
A Compressor insert?
an Aux?
itself? 
It's important to NAME THINGS WHAT THEY ARE OR WHAT THEY DO.
- [ ] revise